contracts/pods/RoleBastion.sol

```diff
@@ -30,9 +30,5 @@ contract RoleBastion is CoreRef {
         require(roleAdmin == bytes32(0), "Role already exists");
         emit BastionRoleCreate(role, TribeRoles.ROLE_ADMIN);
         core().createRole(role, TribeRoles.ROLE_ADMIN);
-
-        // TODO: DAO vote to transfer all roles which have admins of GOVERNOR
-        // VOTIUM, admin GOVERNOR
-        //  transfer
```

Comment:
what are your plans here? Is there a github issue?

Reply:
Implemented in this PR: https://github.com/fei-protocol/fei-protocol-core/pull/678